components/hero/hero.tsx

```diff
@@ -0,0 +1,46 @@
+import Media from '../media';
+import hero from '../../public/images/hero.webp';
+
+import style from './hero.module.scss';
+
+export default function Hero() {
+  return (
+    <div className={style.container}>
+      <div className={style.row}>
+        <div className={style.info}>
+          <h1 className={style.title}>Maintainable data science solved</h1>
```

Comment:
You might want to set the width of this text for the mobile breakpoint as I notice that the width of this text on iphone is longer than it should on the design: 

<img width="553" alt="image" src="https://user-images.githubusercontent.com/27775658/155120782-302752fc-8715-4358-8e02-ae2785c3fb49.png">

<img width="305" alt="image" src="https://user-images.githubusercontent.com/27775658/155120927-48fc908a-9842-4f8d-87f1-2d844ef4e4ca.png">

Reply:
Oh great call. Updated now.